pkg/lightning/.github/challenge-bot.yml

```diff
@@ -1 +0,0 @@
-defaultSigLabel: sig/migrate
```

Comment:
keep this file and move into `/.github/`

Reply:
Sure, thanks!